CMakeLists.txt

```diff
@@ -631,7 +633,7 @@ if( BUILD_PARSEC )
     list(APPEND EXTRA_LIBS ${HWLOC_LIBRARIES})
     list(APPEND EXTRA_INCLUDES ${HWLOC_INCLUDE_DIRS})
     set (PARSEC_PKG_REQUIRE "hwloc")
-    if(MPI_C_FOUND AND NOT CMAKE_CROSSCOMPILING)
+    if(MPI_C_FOUND AND NOT CMAKE_CROSSCOMPILING AND MPI_HWLOC_COMPAT_CHECK)
```

Comment:
I'm not sure if this reads `MPI_C_FOUND AND (NOT CMAKE_CROSSCOMPILING) AND MPI_HWLOC_COMPAT_CHECK` or `MPI_C_FOUND AND NOT (CMAKE_CROSSCOMPILING AND MPI_HWLOC_COMPAT_CHECK)`. I guess CMake does the right thing, but this is CMake... I also guess that could change any time with a policy change ;)

Could we use explicit parentheses here? Or would that be too heavy?